exporter/importables.go

```diff
@@ -953,17 +953,36 @@ var resourcesMap map[string]importable = map[string]importable{
 		Name: func(d *schema.ResourceData) string {
 			return globalWorkspaceConfName
 		},
+		List: func(ic *importContext) error {
+			ic.Emit(&resource{
+				Resource: "databricks_workspace_conf",
+				ID:       globalWorkspaceConfName,
+				Data: ic.Resources["databricks_workspace_conf"].Data(
+					&terraform.InstanceState{
+						ID:         globalWorkspaceConfName,
+						Attributes: map[string]string{},
+					}),
+			})
+			return nil
+		},
 		Import: func(ic *importContext, r *resource) error {
 			wsConfAPI := workspace.NewWorkspaceConfAPI(ic.Context, ic.Client)
 			keys := map[string]any{
-				"enableIpAccessLists":  false,
-				"maxTokenLifetimeDays": 0,
-				"enableTokensConfig":   false,
+				"enableIpAccessLists":                              false,
+				"maxTokenLifetimeDays":                             0,
+				"enableTokensConfig":                               false,
+				"storeInteractiveNotebookResultsInCustomerAccount": false,
+				"maxUserInactiveDays":                              0,
 			}
 			err := wsConfAPI.Read(&keys)
 			if err != nil {
 				return err
 			}
+			for k, _ := range keys {
```

Comment:
it could be simplified to 
```
for k, v := range keys {
 if v == nil {
	delete(keys, k)
 }
}
```

Reply:
won't ti be a concurrent modification? :)

Reply:
According to this SO, no: https://stackoverflow.com/questions/23229975/is-it-safe-to-remove-selected-keys-from-map-within-a-range-loop, especially this answer: https://stackoverflow.com/a/23231539/18627